UI/window-basic-adv-audio.cpp

```diff
@@ -158,35 +156,25 @@ void OBSBasicAdvAudio::SetShowInactive(bool show)
 
 	showInactive = show;
 
-	sourceAddedSignal.Disconnect();
-	sourceRemovedSignal.Disconnect();
-	sourceActivatedSignal.Disconnect();
-	sourceDeactivatedSignal.Disconnect();
+	sigs.clear();
```

Comment:
I just noticed this will clear all signals, even the ones that are only initialized in constructors. I would probably either split it off into two separate vectors and make sure to only clear relevant signals here, or maybe just close the PR.

Need to be careful for things like this. Unless I'm missing something.

Reply:
These are the same signals that are created in the constructor.

Reply:
Ah I see, I misread.